spec/index.bs

```diff
@@ -1267,6 +1267,11 @@ interface GPUAdapter {
     : <dfn>\[[adapter]]</dfn>, of type [=adapter=], readonly
     ::
         The [=adapter=] to which this {{GPUAdapter}} refers.
+
+    : <dfn>\[[preferred_format]]</dfn>, of type {{GPUTextureFormat}}, readonly
```

Comment:
Oh, I guess this needs to be a member on the adapter (multi-thread object) rather than on the GPUAdapter (single-thread object wrapper).

Reply:
This was rendered moot by the decision on the call to have the selection be boiled down to just an algorithm step.